seatunnel-engine/seatunnel-engine-client/src/main/java/org/apache/seatunnel/engine/client/job/ClientJobProxy.java

```diff
@@ -93,21 +89,12 @@ private void submitJob(JobImmutableInformation jobImmutableInformation) {
     @Override
     public JobStatus waitForJobComplete() {
         try {
-            jobResult =
-                    RetryUtils.retryWithException(
```

Comment:
> Why remove retry?

The test found that the retry here is meaningless because once the client and server connection times out, the seaTunnelHazelcastClient will no longer be available, so no matter how many retries are made here, it is useless